client/src/partials/stock/donation_management/donation_management.js

```diff
@@ -199,10 +199,10 @@ var DonationManagementController = function ($q, $translate, $location, $routePa
 
   function validateDonationLine (don){
     var cleaned = true;
-    don.sum = don.lots.data.reduce(function (a, b){ return a + b.quantity}, 0);
+    don.sum = don.lots.data.reduce(function (a, b){ return a + b.quantity; }, 0);
 
     for (var i = don.lots.data.length - 1; i >= 0; i--) {
-      if(don.lots.data[i].lot_number == '' || don.lots.data[i].lot_number == null || don.lots.data[i].quantity <= 0 || don.lots.data[i].expiration_date <= vm.session.date){
+      if(don.lots.data[i].lot_number === '' || don.lots.data[i].lot_number == null || don.lots.data[i].quantity <= 0 || don.lots.data[i].expiration_date <= vm.session.date){
```

Comment:
To make this more readable, can you make this more like:
```js
for (var i = don.lots.data.length - 1; i >= 0; i--) {
  var donation = don.lots.data[i];
  if (donation.lot_number === '' || /* more conditions here */) { }
  /* more code here ... */
}
```
Could you make this change in this PR?